src/data/polynomial/laurent.lean

```diff
@@ -294,6 +294,9 @@ left_inverse_trunc_to_laurent.injective
   f.to_laurent = g.to_laurent ↔ f = g :=
 ⟨λ h, polynomial.to_laurent_injective h, congr_arg _⟩
 
+instance : module R[X] R[T;T⁻¹] :=
+module.comp_hom _ polynomial.to_laurent
```

Comment:
You may as well add the analogous `algebra R[X] R[T;T⁻¹`]` instance for when `R` is commutative, which is no more contentious than this instance

Reply:
I added the instances that I think you meant.  I thought that you said in the other PR that the `algebra R[X] R[T;T⁻¹]` instance was the dodgy one, but maybe I misunderstood.

Anyway, here we go: do you like these?  :smile: